images/installer-origin-release/Dockerfile.ci

```diff
@@ -0,0 +1,12 @@
+# This Dockerfile is used as a CI job to build smoke tests for OpenShift installer
+
+FROM  quay.io/coreos/origin-release:bazel
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+
+ENV USER="bazel"
+ENV HOME="/tmp"
+
+RUN ls -al && \
```

Comment:
We can probably drop the debugging `ls`.

Reply:
yup